.ci/ci_cache_nemu.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+# Copyright (c) 2019 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+# The main purpose of this script is to
```

Comment:
I'm confused tbh. The title of the PR suggests the NEMU binary is being copied *from* the cache *to* it's correct location. But this file only seems to create the cached binary...?